testsuite/classlibrary/TestPatternProxy.sc

```diff
@@ -362,8 +362,31 @@ TestPatternProxy : UnitTest {
 
 		Pdefn.clear;
 		Pdef.clear;
+	}
 
+	test_EventPatternProxy_xfade_cleanup_not_premature {
 
-
+		var assert = { |what, which, important = true|
+			this.assert(what, which, important);
+		};
+		var stream2_cleanup_called = false, p = EventPatternProxy.new.fadeTime_(2), r;
+		p.source = Pbind(\dur, 1.1, \degree, 3, \amp, 0.1);
+		r = p.asStream;
+		// pulling one event is necessary to prime the old stream
+		assert.(r.next(())[\degree] == 3,
```

Comment:
use `assertEquals` for this, and leave `report` as true (the default)

Reply:
I've done as you suggested, but that's a bit verbose in printing...

```
PASS: a TestEventPatternProxy: new - EventPatternProxy xfade 1st source 1st pull shold be executed transparently.
Is:
	 3 
Should be:
	 3
PASS: a TestEventPatternProxy: new - EventPatternProxy xfade 2nd source cleanup should be not called prematurely.
```